apps/homework/views.py

```diff
@@ -105,19 +105,21 @@ def get_context_data(self, **kwargs):
             context['klass'] = submission.klass
             context['logs_possible'] = [
                 'inputfile',
```

Comment:
When this gets formatted on the submission detail page, it looks like `Inputfile`.  The other files are formatted correctly, so this one sticks out to me. Would be awesome if this formatted to `Input File`.

Reply:
I didn't mean to capitalize "File". I'll change that! Thanks dude!